mlir/include/mlir/IR/Builders.h

```diff
@@ -67,6 +67,7 @@ class Builder {
   FloatType getFloat8E4M3FNUZType();
   FloatType getFloat8E4M3B11FNUZType();
   FloatType getFloat8E3M4Type();
+  FloatType getFloat6E3M2FNType();
```

Comment:
nit: it looks like methods are ordered by type bitwidth (?) so I would move this one before the Float8 ones

Reply:
Done, here and in the other places.